packages/app/src/utils/getOrganization.ts

```diff
@@ -24,6 +24,7 @@ const getAsyncOrganization = async (
 ): Promise<Organization> =>
   await client.organization.retrieve({
     fields: {
+      // @ts-expect-error SDK now uses singular fields selection name for singletons but it should stay plural to work properly
```

Comment:
@pviti FYI: As Marco noticed now the SDK is asking to use here the singular version of the `organization` fields key (maybe for its singleton derivation) but the fields selection is still working properly only if the old plural version of the key is used. So I decided to leave it plural and add this `ts-expect-error` to highlight the unexpected behavior.